h/feeds/render.py

```diff
@@ -1,9 +1,20 @@
+import functools
+
 from pyramid import renderers
 
+from h.features import flag_enabled
 from h.feeds import atom
 from h.feeds import rss
 
 
+def annotation_url(request, annotation):
+    """Return the HTML permalink URL for the given annotation."""
+    if flag_enabled(request, 'direct_linking'):
```

Comment:
@nickstenning - I'm guessing we'll want to change how we get the link here completely in light of the discussion around adding a `links` property to annotations.